src/components/existing-request/existing-request-display.vue

```diff
@@ -2,7 +2,7 @@
   <MainContainer id="existing-request-display" class="pa-10">
     <template v-slot:container-header>
       <v-col cols="auto" class="py-0">
-        <span class="h3">{{ nr.nrNum }}</span>
+        <span class="h3 user-select-all">{{ nr.nrNum }}</span>
```

Comment:
This CSS rule allows the user to click on this text and have it all auto-selected. Otherwise, if you double-click on the NR number, the next word is also selected (undesired).

Reply:
Cool!